HIP/hip-830.md

```diff
@@ -0,0 +1,343 @@
+---
+hip: 830
+title: Event Serialization With Round Number, EventDescriptors, and Multiple Other Parents.
+author: Edward Wertz <edward@swirldslabs.com>
+type: Standards Track
+category: Core
+needs-council-approval: Yes
+status: Last Call
+last-call-date-time: 2023-12-04T07:00:00Z
+created: 2023-10-25
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/827
+updated: 2023-11-30
+---
+
+## Abstract
+
+This HIP proposes several changes to the event serialization format.
+
+1. Add the `roundBirth` to the metadata of new events. The `roundBirth` value is 1 + the latest round to have come
+   to consensus at the time the event was created. This values is used to look up the correct consensus
+   roster for validating the event's signature on other nodes.
+2. Represent references to events as `EventDescriptors`. EventDescriptors are used by an event to link
+   to its self-parent and other parents.
+3. Allow for multiple other parents.
+4. In Gossip, reduce an event's unhashed metadata to just the event's signature.
+
+## Motivation
+
+### Add `roundBirth` to Event Metadata
+
+When we allow dynamic address books such that nodes can be added or removed from the consensus roster without restart,
+we need a way to indicate which consensus roster was active when an event was created. A new consensus roster becomes
+active on a round boundary. Writing the `roundBirth` into an event's metadata will provide a way to look up the
+effective consensus roster to validate the event's signature.
+
+The birth round number will also be used to determine when an event has become ancient and is no longer able to come to
```

Comment:
I would switch the order of the statements about signature validation and ancient determination. Leemon made it clear that the ancient determination is the primary reason, so it should be said first.